Detectors/ITSMFT/ITS/simulation/src/Detector.cxx

```diff
@@ -917,8 +917,8 @@ void Detector::constructDetectorGeometry()
 
     LOG(DEBUG1) << "mBuildLevel: " << mBuildLevel[j] << FairLogger::endl;
 
-    if (mStaveThickness[j] != 0) {
-      mGeometry[j]->setStaveThick(mStaveThickness[j]);
+    if (mChipThickness[j] != 0) {
+      mGeometry[j]->setChipThick(mChipThickness[j]);
```

Comment:
The V1Layer class is obsolete and will never be used. Anyway if you think it is better, I can re-implement the deleted member and its Setter/Getter, but they won't be used anywhere.